cve_bin_tool/sbom_manager/__init__.py

```diff
@@ -100,6 +97,41 @@ def get_vendor(self, product: str) -> list:
             vendorlist.append("UNKNOWN")
         return vendorlist
 
+    def parse_sbom(self):
```

Comment:
Can we put a docstring here, since I'm starting to fill those out elsewhere?  Something like "parse SBOM, using PURL identifiers preferentially if found" maybe?